tests/gui_tests/test_gui_methods.py

```diff
@@ -583,6 +584,7 @@ def test_generating_column_data(pytestconfig, test_datastore):
                 "Yemen",
                 "Zambia",
                 "Zimbabwe",
+                "Unknown",
```

Comment:
This is for a separate PR, but Isn't it worth defining this list of countries (plus the list of platform and sensor types) in just one location?  I guess we could have a `tests/consts.py` file where they are centrally defined.
Yes - the rate at which these lists change is forever dropping, and we may not require any future changes, but it seems like a (very small) piece of technical debt.